src/CarbonSystemSolver.jl

```diff
@@ -0,0 +1,1750 @@
+#        Θ     :: FT = 25.0,
+#        Sᴬ    :: FT = 35.0,
+#        Δpᵦₐᵣ :: FT = 0.0,
+#        FT          = Float64,
+        Θᶜ    :: Float64 = 25.0,
+        Sᴬ    :: Float64 = 35.0,
+        Δpᵦₐᵣ :: Float64 = 0.0,
+        )
+
+# Need a conversion from Absolute Salinity, Sᴬ, to Practical Salinity, Sᴾ
+    Sᵖ = Sᴬ
+# Sᴾ = Sᴬ / (1.0 - 0.35 * Sᴬ / 35.0) ??
+# What about converting temperature from Conservative Temperature to potential temperature?
+    Θᴷ = Θᶜ + Θᴷ_0ᵒC
+# Also need to convert from Absolute Pressure, Pᴬ, to Applied Pressure in bars, the pressure relative to (1x) atm
+
+    return (Cᵈⁱᶜₖ₀    = Fᵈⁱᶜₖ₀(Θᴷ, Sᵖ, Pᵈⁱᶜₖ₀),
+            Cᵈⁱᶜₖ₁ᵣ₉₃ = Fᵈⁱᶜₖ₁ᵣ₉₃(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᵈⁱᶜₖ₁ᵣ₉₃),
+            Cᵈⁱᶜₖ₂ᵣ₉₃ = Fᵈⁱᶜₖ₂ᵣ₉₃(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᵈⁱᶜₖ₂ᵣ₉₃),
+            Cᵈⁱᶜₖ₁ₘ₉₅ = Fᵈⁱᶜₖ₁ₘ₉₅(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᵈⁱᶜₖ₁ₘ₉₅)/H⁺ₛoverH⁺ₜ(Θᴷ, Sᵖ, Δpᵦₐᵣ),
+            Cᵈⁱᶜₖ₂ₘ₉₅ = Fᵈⁱᶜₖ₂ₘ₉₅(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᵈⁱᶜₖ₂ₘ₉₅)/H⁺ₛoverH⁺ₜ(Θᴷ, Sᵖ, Δpᵦₐᵣ),
+            Cᵈⁱᶜₖ₁ₗ₀₀  = Fᵈⁱᶜₖ₁ₗ₀₀(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᵈⁱᶜₖ₁ₗ₀₀),
+            Cᵈⁱᶜₖ₂ₗ₀₀  = Fᵈⁱᶜₖ₂ₗ₀₀(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᵈⁱᶜₖ₂ₗ₀₀),
+            Cᵇₖ₁      = Fᵇₖ₁(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᴮₖ₁),
+            Cᴴ²ᴼₖ₁    = Fᴴ²ᴼₖ₁(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᴴ²ᴼₖ₁)/H⁺ₛoverH⁺ₜ(Θᴷ, Sᵖ, Δpᵦₐᵣ),
+            Cᴾᴼ⁴ₖ₁    = Fᴾᴼ⁴ₖ₁(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᴾᴼ⁴ₖ₁)/H⁺ₛoverH⁺ₜ(Θᴷ, Sᵖ, Δpᵦₐᵣ),
+            Cᴾᴼ⁴ₖ₂    = Fᴾᴼ⁴ₖ₂(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᴾᴼ⁴ₖ₂)/H⁺ₛoverH⁺ₜ(Θᴷ, Sᵖ, Δpᵦₐᵣ),
+            Cᴾᴼ⁴ₖ₃    = Fᴾᴼ⁴ₖ₃(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᴾᴼ⁴ₖ₃)/H⁺ₛoverH⁺ₜ(Θᴷ, Sᵖ, Δpᵦₐᵣ),
+            Cˢⁱᵗₖ₁    = Fˢⁱᵗₖ₁(Θᴷ, Sᵖ, Pˢⁱᵗₖ₁)/H⁺ₛoverH⁺ₜ(Θᴷ, Sᵖ, Δpᵦₐᵣ),
+            Cᴴ²ˢₖ₁    = Fᴴ²ˢₖ₁(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᴴ²ˢₖ₁)/H⁺ₛoverH⁺ₜ(Θᴷ, Sᵖ, Δpᵦₐᵣ),
+            Cᴺᴴ⁴ₖ₁    = Fᴺᴴ⁴ₖ₁(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᴺᴴ⁴ₖ₁)/H⁺ₛoverH⁺ₜ(Θᴷ, Sᵖ, Δpᵦₐᵣ),
+            Cᴴᶠᵦ₁     = Fᴴᶠᵦ₁(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᴴᶠᵦ₁),
+            Cᴴᶠₖ₁     = Fᴴᶠₖ₁(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᴴᶠₖ₁),
+            Cᴴˢᴼ⁴ₖ₁   = Fᴴˢᴼ⁴ₖ₁(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᴴˢᴼ⁴ₖ₁),
+            Cᶜᵃˡᶜⁱᵗᵉₛₚ = Fᶜᵃˡᶜⁱᵗᵉₛₚ(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᶜᵃˡᶜⁱᵗᵉₛₚ),
+            Cᵃʳᵃᵍᵒⁿⁱᵗᵉₛₚ = Fᵃʳᵃᵍᵒⁿⁱᵗᵉₛₚ(Θᴷ, Sᵖ, Δpᵦₐᵣ, Pᵃʳᵃᵍᵒⁿⁱᵗᵉₛₚ),
+            Cᴮᵀ         = Bᵀᴼᵀ(Sᵖ),
+            Cᶠᵀ         = Fᵀᴼᵀ(Sᵖ),
+            Cᶜᵃ         = Caᵀᴼᵀ(Sᵖ),
+            Cˢᴼ⁴        = SO₄ᵀᴼᵀ(Sᵖ),
+            CH⁺ₛoverH⁺ₜ  = H⁺ₛoverH⁺ₜ(Θᴷ, Sᵖ, Δpᵦₐᵣ),
+            CH⁺ₜoverH⁺₃ = H⁺ₜoverH⁺₃(Θᴷ, Sᵖ, Δpᵦₐᵣ),
+            CH⁺ₛoverH⁺₃ = H⁺ₛoverH⁺₃(Θᴷ, Sᵖ, Δpᵦₐᵣ),
+    )
+    end # end function
+end # end struct
+
+# Gas constant
+const gasconst_bar_cm3_o_mol_k = 83.14472 # Handbook (2007)
+
+# 0 degrees centigrade in Kelvin
+const Θᴷ_0ᵒC = 273.15 # Handbook (2007)
+
+@inline Θᴷ₁₀₀(Θᴷ) = Θᴷ/100
+@inline Sᴾ⁰⁵(Sᵖ)  = sqrt(Sᵖ)
+@inline ΘᵒC(Θᴷ)   = Θᴷ - Θᴷ_0ᵒC
+@inline ΔSᵖ(Sᵖ)   = Sᵖ - 34.8
+@inline Rₜ(Θᴷ)     = gasconst_bar_cm3_o_mol_k * Θᴷ
+
+@inline """
+    H₂Oˢʷ(Sᵖ)
+
+    # Function returns the mass of pure water in one kg of seawater
+    # of salinity s
+    # References: "libthdyct" -- derived by Munhoven (1997) from data by Millero (1982)
+    #             "Handbook (2007)" -- Handbook (2007)
+    # pH scale:   N/A
```

Comment:
```suggestion
    H₂Oˢʷ(Sᵖ)

Return the mass of pure water in one kilogram of seawater with salinity `Sᵖ`.

References
=========
* "libthdyct" -- derived by Munhoven (1997) from data by Millero (1982)
* "Handbook (2007)" -- Handbook (2007)
* pH scale:   N/A
```

This fixes a few formatting and style issues --- do not indent the lines of the docstring or use comment symbols (#), use imperative form, code (ie the function arguments) should be surrounded with single ticks `likethis`.

Check this out for more

https://docs.julialang.org/en/v1/manual/documentation/#Writing-Documentation